doc/users/dflt_style_changes.rst

```diff
@@ -473,6 +473,37 @@ or by setting::
 in your :file:`matplotlibrc` file.
 
 
+``fill_between`` and ``fill_betweenx``
+--------------------------------------
+
+`~matplotlib.axes.Axes.fill_between` and
+`~matplotlib.axes.Axes.fill_betweenx` both follow the patch color
+cycle.
+
+.. plot::
+
+   import matplotlib.pyplot as plt
+   import numpy as np
+
+   fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(6, 3))
+   th = np.linspace(0, 2*np.pi, 128)
+   N = 5
+
+   def demo(ax, extra_kwargs, title):
+       ax.set_title(title)
+       return [ax.fill_between(th, np.sin((j / N) * np.pi + th), alpha=.5, **extra_kwargs)
+               for j in range(N)]
+
+   demo(ax1, {}, '2.x')
+   demo(ax2, {'facecolor': 'C0'}, 'non-cycled')
+
+If the facecolor is set via the ``facecolors`` or ``color`` keyword argument,
```

Comment:
The two arguments `facecolor` and `facecolors` exists for this function, but it'd be nice if it was consistently used on this example. You use `facecolor` above but then mention `facecolors` in the text. Can you use only one?